docs/validators/snapshots_archives.md

```diff
@@ -0,0 +1,29 @@
+# Snapshots
+
+Quickly sync your node with Evmos network
+
+## Public Available Endpoints
+
+Below is a list of publicly available snapshots that you can use to sync with the Evmos mainnet and
+archived 9001-1 mainnet:
+
+<!-- markdown-link-check-disable -->
+:::: tabs
+::: tab Snapshots
+
+| Name       | Url                                                                     |
+| -----------|------------------------------------------------------------------------ |
+| `Staketab` | `https://github.com/staketab/nginx-cosmos-snap/blob/main/docs/evmos.md` |
+| `Polkachu` | `https://www.polkachu.com/tendermint_snapshots/evmos`                   |
+:::
+::: tab Archives
+<!-- markdown-link-check-disable -->
+
+| Name           | Url                                                                             |
+| ---------------|---------------------------------------------------------------------------------|
+| `genesis.json` | `https://archive.evmos.dev/genesis/genesis.json`                                |
+| `Nodes Guru`   | `https://snapshots.nodes.guru/evmos_9001-1/`                                    |
+| `Polkachu`     | `https://www.polkachu.com/tendermint_snapshots/evmos`                           |
+| `Forbole`      | `https://s3.bigdipper.live.eu-central-1.linodeobjects.com/evmos_9001-1.tar.lz4` |
```

Comment:
some of these are genesis JSON files, while others are snapshots for the entire chain